sync/src/types.rs

```diff
@@ -507,6 +514,69 @@ impl HeaderView {
     pub fn into_inner(self) -> Header {
         self.inner
     }
+
+    pub fn build_skip<F>(&mut self, mut get_header_view: F)
+    where
+        F: FnMut(&H256) -> Option<HeaderView>,
```

Comment:
Why need `FnMut`?

Reply:
Could be removed.

Reply:
Could be just `Fn`

Reply:
If we want test this method (measure steps for example) in the future, we need `FnMut`